galley/pkg/config/source/kube/apiserver/status/util.go

```diff
@@ -26,12 +24,12 @@ func toStatusValue(msgs diag.Messages) interface{} {
 		return nil
 	}
 
-	var lines strings.Builder
-
+	result := make([]interface{}, 0)
 	for _, m := range msgs {
-		lines.WriteString(m.StatusString())
-		lines.WriteString("\n")
+		// For the purposes of status update, the origin field is redundant
+		// since we're attaching the message to the origin resource.
+		result = append(result, m.Unstructured(false))
```

Comment:
Do you ever need the origin field in Unstructured? Is it for the istioctl JSON output case?

Reply:
That's what I had in mind, yes. 

I also considered something like `m.Unstructured().WithoutOrigin()` to avoid the awkward boolean flag, but thought it probably wasn't worth it.